app/validators/profile_validator.rb

```diff
@@ -16,7 +16,10 @@ def validate(record)
     # NOTE: The summary is a base profile field, which we add to all new Forem
     # instances, so it should be safe to validate. The method itself also guards
     # against the field's absence.
-    record.errors.add(:summary, I18n.t("validators.profile_validator.too_long")) if summary_too_long?(record)
+    if summary_too_long?(record)
+      record.errors.add(:summary,
+                        message: I18n.t("validators.profile_validator.too_long"))
```

Comment:
**Suggestion:** The problem with passing in `:bio` here is that there isn't a method/attribute with that name and passing in `:summary` will continue to display the incorrect error message.

Like [you mentioned here](https://github.com/forem/forem/issues/13119#issuecomment-1334176562) the attribute is called `summary` in the DB and ActiveRecord won't allow us to do something like `record.errors.add(:bio, ...`

So in that case we can use `:base` instead so the validation isn't strictly linked to a model attribute. I believe something like this would work:

```suggestion
      record.errors.add(:base,
                        message: I18n.t("validators.profile_validator.bio_too_long"))
```

Notice that in this case I also changed the translation key to `bio_too_long` (so it's named in a more clear way).

You'll have to change the key in both in [`config/locales/validators/en.yml`](https://github.com/forem/forem/blob/bec3edc756f12d7c7579f8119bf1d3e0a1733876/config/locales/validators/en.yml#L13) and [`config/locales/validators/fr.yml`](https://github.com/forem/forem/blob/bec3edc756f12d7c7579f8119bf1d3e0a1733876/config/locales/validators/fr.yml#L13) and also the content of the translation to `"Bio is too long"`

Reply:
Whoops, sorry I also just properly read through @jaw6's [other suggestion](https://github.com/forem/forem/pull/18798#discussion_r1038032351) on how to rename the summary text to bio. I guess either could work to rename the error message 🙂